pkgs/development/python-modules/httpagentparser/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "httpagentparser";
+  version = "1.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "73Y9MZk912GCWs7myLNL4yuVzxZ10cc8PNNfnlKDGyY=";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  # PyPi version does not include test directory
```

Comment:
Perhaps upstream starts to tag the source.

https://github.com/shon/httpagentparser/issues/88

Reply:
thanks for opening the issue upstream!